### PR TITLE
39 fluid units

### PR DESCRIPTION
**Pull request by jhogue**
Add Fluid-Units to our mixin library, with comments and examples on how it can get used. Simplify the typographic sizes. Apply to our existing typography elements. 

Reorganize the mixin folder to be more discrete and use smaller files.

### CODE REVIEW

**Comment by jhogue** [comment]
Review here please? Someone? Bueller? 

@johncionci @kbeck303 @syzygy333 @BrandonDH

File: base/_typesizes.scss [hunk 1]
```diff
@@ -1,158 +1,43 @@
...
-// The idea:
-//  • One list of breakpoints.
-//  • One list of type sizes/scales with a corresponding key name.
-//
-// Helpers:
-//  • @include all-type-sizes( size-grouping ): A mixin that can iterate and output all type sizes from one breakpoint
-//  • @incude one-element-size( element ): A mixin that can iterate over the breakpoints, find the
-//	  matching type size, and output all breakpoint sizes for that element.
-// • type-size( element, size-grouping ): A function that can output one type size at one specific breakpoint
+// Required list of sizes for typographics elements
```
Comment (kbeck303) [inline base/_typesizes.scss]: Should use the same comment style as this page: https://github.com/oomphinc/scss-scaffold/blob/master/base/_mixins.scss

File: base/_typesizes.scss [hunk 2]
```diff
@@ -1,158 +1,43 @@
...
-		giga: 40,
-		mega: 35,
-		kilo: 31,
-		h1: 27,
-		h2: 24,
-		h3: 21,
-		h4: 19,
-		base: 17,
 		micro: 14,
+		base: 16,
```
Comment (kbeck303) [inline base/_typesizes.scss]: indent consistency

File: base/mixins/_main.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+////
+// Oomph Scaffold Mixins
```
Comment (kbeck303) [inline base/mixins/_main.scss]: /// for consistency

**Review by kbeck303 — COMMENTED** [review]
base/mixins/.DS_Store - .DS_Store should be in the .gitignore file

@jhogue I added a few comments

**Comment by jhogue** [comment]
I used this file-level annotation block style for the big ones. Those are meant to encapsulate attributions for all linked asset files. They are the main block style… http://sassdoc.com/file-level-annotations/

As I understand it, any declarations in that block are also used for linked blocks unless that declaration is present at a local level (it would override the global declaration). 

I can't seem to find the DS_Store file that you mention. I don't see its locally in my Terminal and it was already in the `gitignore` file.

**Comment by johncionci** [comment]
@jhogue pretty sure thats a "hidden" file, I do see it as well. In the `.gitignore` we need to add:
`.DS_Store` above the other two declarations.

**Comment by jhogue** [comment]
I guess this line doesn't work? `*/.DS_Store`?

**Comment by johncionci** [comment]
Nope

**Comment by jhogue** [comment]
Removed. Approved?

**Comment by kbeck303** [comment]
yep

/approved

**Pull request merged**